deploy/agent-vm.mk

```diff
@@ -10,14 +10,15 @@ agent-vm: bin/output/qcow2/disk.qcow2
 	sudo cp bin/output/qcow2/disk.qcow2 $(VMDISK)
 	sudo chown libvirt:libvirt $(VMDISK) 2>/dev/null || true
 	sudo virt-install --name $(VMNAME) \
+					  --tpm backend.type=emulator,backend.version=2.0,model=tpm-tis \
 					  --vcpus $(VMCPUS) \
 					  --memory $(VMRAM) \
 					  --import --disk $(VMDISK),format=qcow2 \
 					  --os-variant fedora-eln  \
 					  --autoconsole text \
 					  --wait $(VMWAIT) \
 					  --transient || true
-
+	sudo firewall-cmd --zone=libvirt --add-port=7443/tcp
```

Comment:
Is firewall-cmd the default for all linux distros?

Reply:
Also, tweaking firewall rules when the cluster is up will break the network settings and tweaks made by podman on the kernel network stack (because of how firewalld works...). 

I would put any necessary commands to open ports as part of the configuration pre-requisites (documentation). Would that make sense?